### PR TITLE
Mapped a few entity/item/player stuff. 

### PR DESCRIPTION
**Pull request by Darkhax**
Looks like there is a merge conflict, crap x)

### CODE REVIEW

File: mappings/net/minecraft/entity/boss/DragonManager.mapping [hunk 1]
```diff
@@ -0,0 +1,2 @@
+CLASS axc net/minecraft/entity/boss/DragonManager
+	METHOD e resurectDragon ()V
```
Comment (exaskye) [inline mappings/net/minecraft/entity/boss/DragonManager.mapping]: Misspelled; `resurrectDragon`

File: mappings/net/minecraft/item/ItemDoor.mapping [hunk 2]
```diff
@@ -1,3 +1,11 @@
 CLASS agh net/minecraft/item/ItemDoor
 	FIELD a block Lank;
 	METHOD a activate (Lacu;Lali;Ldx;Lsw;Lee;FFF)Lsy;
+		ARG 0 player
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 and
```
Comment (exaskye) [inline mappings/net/minecraft/item/ItemDoor.mapping]: `and`? Should this be `hand`?

File: mappings/net/minecraft/item/ItemMap.mapping [hunk 3]
```diff
@@ -1,3 +1,6 @@
 CLASS age net/minecraft/item/ItemMap
 	METHOD a createMapPacket (Lahf;Lali;Lacu;)Lgv;
+		ARG 0 stack
+		ARG 1 wrold
```
Comment (exaskye) [inline mappings/net/minecraft/item/ItemMap.mapping]: `world`

**Review by exaskye — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/entity/boss/DragonManager.mapping (hunk 1 above)
Comment (Darkhax) [reply mappings/net/minecraft/entity/boss/DragonManager.mapping]: Thanks for spotting these.

**Review by exaskye — APPROVED** [review]

**Pull request merged**